src/main/java/com/redhat/cloud/notifications/events/EndpointProcessor.java

```diff
@@ -60,6 +64,7 @@ void init() {
                     return endpointTypeToProcessor(endpoint.getType()).process(endpointNotif);
                 })
                 .merge()
+                // Todo: Remove this filtering once we are ready to send emails too
```

Comment:
Forgot?

Reply:
Yes. Totally forgot about it.
I was going to have a hell of a time testing it, thanks for noticing!!